resetpassword.html

```diff
@@ -1,11 +1,37 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Reset password</title>
-</head>
-<body>
-    
-</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link href="style/resetpassword.css" rel="stylesheet">
+    <script src="script/global.js" defer></script>
+    <title>Reset password</title>
+</head>
+<body>
+    <div class="reset-container">
+        <form>

```

Comment:
If the form is being used to send data to backend, make sure that the form has its action as POST. E.g. <form action="POST">...</form>